packages/router-devtools/package.json

```diff
@@ -72,6 +72,7 @@
   },
   "peerDependencies": {
     "@tanstack/react-router": "workspace:^",
+    "@tanstack/react-router-devtools": "workspace:^",
```

Comment:
Should this be `peerDependency`? or a `dependency`?

Since the user wouldn't be installing both `@tanstack/router-devtools` + `@tanstack/react-router-devtools`.

Reply:
That's probably best yes. I've moved it to deps